### PR TITLE
(lha|phpbrew): update 

### PR DESCRIPTION
**Pull request by SMillerDev**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Accompanies https://github.com/Homebrew/homebrew-core/pull/159061

### CODE REVIEW

File: Formula/l/lha.rb [hunk 1]
```diff
@@ -1,22 +1,11 @@
 class Lha < Formula
   desc "Utility for creating and opening lzh archives"
-  homepage "https://lha.osdn.jp/"
-  # Canonical: https://osdn.net/dl/lha/lha-1.14i-ac20050924p1.tar.gz
-  url "https://dotsrc.dl.osdn.net/osdn/lha/22231/lha-1.14i-ac20050924p1.tar.gz"
-  version "1.14i-ac20050924p1"
-  sha256 "b5261e9f98538816aa9e64791f23cb83f1632ecda61f02e54b6749e9ca5e9ee4"
+  homepage "https://github.com/jca02266/lha/"
+  url "https://github.com/jca02266/lha/archive/refs/tags/release-20211125.tar.gz"
+  version "1.14i-ac20211125"
```
Comment (MikeMcQuaid) [inline Formula/l/lha.rb:5]: where does the `1.14i-ac` bit come from?

**Review by MikeMcQuaid — APPROVED** [review]
One comment, not blocking, feel free to answer post-merge.

File: Formula/l/lha.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/l/lha.rb:5]: Part of the tag message:
> LHa for UNIX version 1.14i-ac20211125 (x86_64-w64-mingw32)

Comment (MikeMcQuaid) [reply Formula/l/lha.rb:5]: @SMillerDev Cool, good to ship then!

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request merged**